recipes/piqs/meta.yaml

```diff
@@ -0,0 +1,65 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "piqs" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "13c814410228d598dddc28f842ddbea2429f821f059a8ef1695731abb02afffd" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nathanshammah/piqs/tree/master/dist/{{ name }}-{{ version }}.tar.gz
```

Comment:
you should use a release link, like this one:

https://github.com/nathanshammah/piqs/archive/{{ version }}.tar.gz

with its SHA256:

b1f17e3190f4a8bdb54d24b772fe4a6dfd9e093a83402508766ed47865d74b76

Reply:
Hi, thanks for the help. I supposed I do not have to archive the repo and simply create a new release on Github for this to work? I am making a new release in https://github.com/nathanshammah/piqs/pull/58

Reply:
you don't need to release again, but now that you did, just bump the `version` variable to 1.2 and use the sha256 of the 1.2 release